markdown/3.0/en/ff-header-navigation.md

```diff
@@ -254,22 +103,101 @@ You can also set custom content after each group. Just add an element with the g
 This will add the content under the group items of the 'Outdoor jackets' group.
 ![navigation-group-slot.png](../../images/doku/navigation-group-slot.png "navigation group slot")
 
-### mixins
 
-They will be inserted into the layout and you can style them with basic css classes. The layout is `flex`
-based and if you really want to, you can also modify that behavior with mixins:
+## Rendered HTML
+
+For this element, the structure of the elements inside the navigation is fixed. You define the appearance of the navigation only based on CSS. The following is an example of what `ff-header-navigation` could render.
+
+```html
+<ff-header-navigation>
```

Comment:
[optional] reduce long code snippet by removing empty lines and repetitive parts.

Reply:
Removed a few of the repetitive things. The empty lines are actually intended to provide some visual clues to logic units. Like the group-caption, nav-links and the group slot.